mintlify/openapi.yaml

```diff
@@ -0,0 +1,4453 @@
+openapi: 3.1.0
```

Comment:
Is it easy enough to make this a symlink instead so that we don't have to worry about copying it over?

Reply:
Oh hmm... Yeah want to give it a shot? Otherwise not the end of the world, just not ideal.